src/SlugUpdateHandler/ProductTranslationSlugUpdateHandler.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusRedirectPlugin\SlugUpdateHandler;
+
+final class ProductTranslationSlugUpdateHandler extends SlugUpdateHandler
+{
+    protected function generateUrl(string $slug): string
```

Comment:
I would rather make it private since class is final.

Reply:
That is not possible since it inherits from parent

Reply:
Oh yes indeed. missed it